scripts/check_ecosystem.py

```diff
@@ -366,7 +366,7 @@ async def limited_parallelism(coroutine: T) -> T:
                 for line in diff_str.splitlines():
                     # Find rule change for current line or construction
                     # + <rule>/<path>:<line>:<column>: <rule_code> <message>
-                    matches = re.search(r": ([A-Z]{1,3}[0-9]{3,4})", line)
+                    matches = re.search(r": ([A-Z]{1,4}[0-9]{3,4})", line)
```

Comment:
What about `: ([A-Z]+[0-9]+)`? This should be more future proof and i'm away of any cases where this would cause false positives

Reply:
Yeah, we could do that. I can't think of any cases either.